### PR TITLE
create setSpecies method

### PR DESCRIPTION
**Pull request by hailzrene**

### CODE REVIEW

**Comment by hailzrene** [comment]
create getLastFeedingTime method

File: src/main/java/edu/kcc/animal/Animal.java [hunk 1]
```diff
@@ -55,6 +55,9 @@ public String getSpecies(){
     }
     
     // TODO: Need setSpecies method (Haley)
+    public void setSpecies(){
```
Comment (mlhaus) [inline src/main/java/edu/kcc/animal/Animal.java:58]: Missing parameter

**Review by mlhaus — CHANGES_REQUESTED** [review]
Please see my inline comments to make corrections.

**Review by hailzrene — COMMENTED** [review]
added parameters

**Pull request merged**